sermant-agentcore/sermant-agentcore-core/src/main/java/io/sermant/core/event/EventCollector.java

```diff
@@ -41,7 +41,7 @@ public class EventCollector {
     // the queue is full, event is reported automatically
     private final BlockingQueue<Event> eventQueue = new ArrayBlockingQueue<>(100);
 
-    private final EventConfig eventConfig = ConfigManager.getConfig(EventConfig.class);
+    private EventConfig eventConfig = ConfigManager.getConfig(EventConfig.class);
```

Comment:
Change private to protected so that subclasses can use it.

Reply:
done